flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/health/DummyHealthProbe.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.kubernetes.operator.health;
+
+/** Health probe that always returns healthy once the operator has successfully started. */
+public class DummyHealthProbe implements HealthProbe {
```

Comment:
It's a bit weird to call it dummy even though it always returns true.

Reply:
It's "dummy" because it contains no logic inside. It's always true :)

Reply:
I will try to improve the naming with some other simplifications